questions-js1/2.md

```diff
@@ -5,5 +5,6 @@ console.log(2 * 4 + 10 / 5);
 
 A: {YOUR ANSWER HERE}
 
+It will print: 10
```

Comment:
It might be pedantic but I think the answer should be added inside a curly bracket.